### PR TITLE
Story section 

### PR DESCRIPTION
**Pull request by jennymakki**
Description:
I have styled and implemented images to the story-section.
Issue #22 

Type of change:
- Responsive grid that stacks vertically in mobile version.
- Added images.
- Small changes in the HTML-section.

Additional information: 
none

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -92,28 +92,24 @@ <h2 class="tagline__heading">KINO - EN ALLDELES SPECIELL BIOUPPLEVELSE</h2>
...
             doloribus nobis exercitationem sunt, dolore quisquam nesciunt ipsa. Ratione maxime, officiis facilis velit
             ab odio vel.
           </p>
         </div>
-        <img src="image2.jpg" alt="Image of a hallway leading to a door with the Kino logo" class="story__image" />
-      </div>
+        <img src="images/Kino_doors.png" alt="Image of a hallway leading to a door with the Kino logo" class="story__image_2" />
     </section>
     <section class="event">
       <a href="#"><img src="image1" alt="Image of an event" /></a>
```
Comment (alfredbrannare) [inline index.html:115]: Everything looks good but this placeholder is showing. I guess we can remove it/change it in another issue.
image

**Review by alfredbrannare — APPROVED** [review]
See comment on line 115 in index.html

Otherwise it looks good!

File: index.html (hunk 1 above)
Comment (jennymakki) [reply index.html:115]: Oh thank you for pointing that out! I had missed to remove that. I can fix that.

**Review by Klumpish — APPROVED** [review]
looks good, nice images.

**Pull request merged**